packages/js/product-editor/src/blocks/generic/checkbox/block.json

```diff
@@ -38,5 +42,7 @@
 		"__experimentalToolbar": false
 	},
 	"editorStyle": "file:./editor.css",
-	"usesContext": [ "postType" ]
-}
+	"usesContext": [
+		"postType"
+	]
+}
```

Comment:
Is it intentional that the ending blank line has been removed in several `block.json` files?

Reply:
I think it's auto formatted by editor. The formatting config is different from mine btw. @senadir can you confirm?

We don't have a configuration for JSON, which I think we should add one, using prettier.

Reply:
Yeah that was auto formatted.